log messages to file: C:\NatlinkGIT3\Natlink\PyTest\testresult.txt
starting unittestNatlink

starting tests with threading: 0

testParser
testGrammar

--------------- errors -----------------

---------- testParser (__main__.UnittestNatlink) --------
Traceback (most recent call last):
  File "<string>", line 2136, in testParser
  File "<string>", line 2110, in checkExperiment
TestError: Unexpected result for GrammarBase.sawBegin
  Expected 1
  Saw 0

--------------- failures -----------------
